manim/mobject/opengl_mobject.py

```diff
@@ -1889,9 +1882,7 @@ def set_rgba_array(self, color=None, opacity=None, name="rgbas", recurse=True):
         return self
 
     def set_rgba_array_direct(self, rgbas: np.ndarray, name="rgbas", recurse=True):
-        """Directly set rgba data from `rgbas` and optionally do the same recursively
-        with submobjects. This can be used if the `rgbas` have already been generated
-        with the correct shape and simply need to be set.
+        """Directly set rgba data from `rgbas` and optionally do the same recursively with submobjects. This can be used if the `rgbas` have already been generated with the correct shape and simply need to be set.
```

Comment:
What flake error does this fix? It seems to cause long lines, which is undesirable too.

Reply:
That is a poorly written docstring. The docstring standard states you must write a one-line summary.